test/components/tour-table.spec.tsx

```diff
@@ -61,7 +61,7 @@ describe('tour-table component test', () => {
   });
   it('handles click on delete tour button', () => {
     const wrapper2 = shallow<TourTable>(<TourTable
-      tour={[{ _id: '123' }]}
+      tour={[['{ _id: "123" }']]}
```

Comment:
@Unesdala these here are incorrect, it should be an array of the Song type, please find where you made it an array of strings and fix, thanks